splunk-otel-android/src/main/java/com/splunk/rum/RumResponseAttributesExtractor.java

```diff
@@ -41,7 +41,7 @@ protected void onStart(AttributesBuilder attributes, Request request) {
     }
 
     @Override
-    protected void onEnd(AttributesBuilder attributes, Request request, Response response) {
+    protected void onEnd(AttributesBuilder attributes, Request request, Response response, Throwable error) {
```

Comment:
@jkwatson now that `AttributesExtractor` accepts error param we should probably call `SplunkRum.addExceptionAttributes()` in this extractor (same as it's done in `OkHttpRumInterceptor`) - I can implement this in another PR.

Reply:
yes!